doc/script_commands.txt

```diff
@@ -2260,6 +2260,31 @@ array, shifting all the elements beyond this towards the beginning.
 
 ---------------------------------------
 
+*countinarray <array name>{[<start index>]},<value1>{,<value2>...<valueN>};
+
+This command will check for matches between the array values and return the number of matches.
+While being optional, if [<start index>] is supplied, the search will begin from the given index value.
```

Comment:
i still need to receive the index and make `i` start from it

also i am thinking of making the other array command we talk about here in one PR
`countinarray` the command here
`inarray` accept 1 arg and return the index of the value while true and return -1 while false